_posts/2024-07-22-kubeflow-1.9-release.md

```diff
@@ -0,0 +1,226 @@
+
+We are looking for new members who can help us craft complete and high quality documentation. Please get involved by [opening and reviewing PRs](https://github.com/kubeflow/website/pulls) in the Kubeflow website.
+
+## Honorable Mentions
+
+### Google Spark Operator migration to Kubeflow
+
+We’re excited to announce the migration of Google’s Spark Operator to the Kubeflow Spark Operator, marking the launch of a significant addition to the Kubeflow ecosystem. The Kubeflow Spark Operator simplifies the deployment and management of Apache Spark applications on Kubernetes. This announcement isn’t just about a new piece of technology, it’s about building a stronger, open-governed, and more collaborative community around Spark on Kubernetes.
+
+Kubeflow Spark Operator is not yet officially included in the Kubeflow release, but you can install it by following the instructions [here](https://www.kubeflow.org/docs/components/spark-operator/getting-started/).
+
+Read more about Kubeflow Spark operator in the announcement [blog post](https://blog.kubeflow.org/operators/2024/04/15/kubeflow-spark-operator.html).
+
+### Google Summer of Code
+
+This year, Kubeflow was excited to participate in Google Summer of Code (GSoC), attracting a wave of enthusiastic students! Over 250 students joined our Slack channel, eager to learn about contributing to the Kubeflow community and crafting impactful proposals.
+
+We were also fortunate to have a dedicated group of 20 mentors ready to guide these talented individuals. From a pool of nearly 70 proposals, we have selected 10 proposals and were awarded with 9 outstanding students by Google. They are now actively contributing to various Kubeflow features, making a real difference in various Kubeflow components.
+
+We'll be following their progress and sharing their accomplishments through a series of blog posts in the future, so stay tuned! A big thank you to all the mentors and students who are making Kubeflow's GSoC 2024 a huge success!
+
+## What’s next
+
+The community continues to see growth, especially with the ever growing interest from the CloudNative community in AI topics. We have recently elected the first Kubeflow Steering Committee. This is the first step towards a more mature governance structure and a democratic and open community.
+
+If you want to take a peek into the Kubeflow 1.10 roadmap planning and contribute with your ideas, see [Notebooks](https://github.com/kubeflow/kubeflow/issues/7459), [Manifests & Security](https://github.com/kubeflow/manifests/issues/2763), [Pipelines](https://github.com/kubeflow/pipelines/discussions/10908), Model Registry, Katib, Training Operator. 
```

Comment:
@tarilabs  please provide a link to the MR roadmap for the next version

@andreyvelich @johnugeorge @tenzen-y please provide links to Katib and Training Operator roadmaps for the next versions.